reconcile/gitlab_permissions.py

```diff
@@ -26,6 +27,25 @@ def get_members_to_add(repo, gl, app_sre):
     return members_to_add
 
 
+def get_all_app_sre_maintainers(repo, gl, app_sre):
+    app_sre_user_ids = [user.id for user in app_sre]
+    app_sre_maintainers = []
+    chunks = [
+        app_sre_user_ids[i : i + PAGE_SIZE]
+        for i in range(0, len(app_sre_user_ids), PAGE_SIZE)
+    ]
+    for chunk in chunks:
+        maintainers = gl.get_project_maintainers(
+            repo, query=create_user_ids_query(chunk)
+        )
+        app_sre_maintainers.extend(maintainers)
+    return app_sre_maintainers
+
+
+def create_user_ids_query(ids):
+    return {"user_ids": ",".join(str(id) for id in ids)}
```

Comment:
according to [api doc](https://docs.gitlab.com/ee/api/members.html#list-all-members-of-a-group-or-project-including-inherited-and-invited-members), `user_ids` is array of ints, have you tried to pass it directly like `{"user_ids": ids}`

Reply:
@hemslo  by passing it directly it is taking the value as multilpe `user_ids` in api like, 
`?user_ids=xxx&user_ids=xx1&user_ids=xx2` which results the API to return only the user having the last user_id field in url query. But if we convert it to above format it creates a coma seperated query 
`?user_ids=xxx,xx1,xx2` and returns all the users matching the ids mentioned